pngpaste.m

```diff
@@ -9,6 +9,8 @@
 {
     fprintf(stderr,
         "Usage: %s [OPTIONS] <dest.png>\n"
+        "\t-\t" "Print to standard output" "\n"
```

Comment:
Hmm good addition, though technically I should probably change the description around a bit since this would be in place of `<dest.png>`...